tests/static/intentManager/printing-intent.xml

```diff
@@ -24,7 +24,7 @@
 		</model>
 		<odkx:intent xmlns:odkx="http://opendatakit.org/xforms" id="print_data" class="org.commcare.dalvik.action.PRINT">
 			<extra key="text" ref="/data/text" />
-			<extra key="cc:print_template_reference" ref="jr://file/commcare/doc/data/print_data.doc" />
+			<extra key="cc:print_template_reference" ref=".">jr://file/commcare/doc/data/print_data.doc</extra>
```

Comment:
`ref` can only be an xpath and can't actually contain the literal of the file name

Block on @ctsims saying this is ok though.